03LessonThree/homework/git-scm.html

```diff
@@ -50,11 +50,11 @@
             <p>It is a JavaScript runtime built on Chrome's V8 JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node.js' package ecosystem, npm, is the largest ecosystem of open source libraries in the world.</p>

```

Comment:
Kira, while inserting the text as-is works _most_ of the time, there are certain characters that have special representations; the ® symbol is one such character. It's better to use `&reg;` (which will render as the registered sign). See http://www.w3schools.com/html/html_entities.asp for more explanation.